hack/make-rules/test-integration.sh

```diff
@@ -85,6 +85,7 @@ checkEtcdOnPath() {
   which etcd && return
   kube::log::status "Cannot find etcd, cannot run integration tests."
   kube::log::status "Please see docs/devel/testing.md for instructions."
```

Comment:
Thanks Christoph! Will update the link.

I still would like to add `You can use 'hack/install-etcd.sh' to install a copy in third_party/.` here. Suppose that if there is only a link here, developers should 1) copy the link and visit it in browser, 2) find `hack/install-etcd.sh` on the page, 3) copy command `hack/install-etcd.sh` back to shell and run it, 4) run extra `export` step. But if we have `You can use 'hack/install-etcd.sh' to install a copy in third_party/.`, developers only need to copy&paste the command and run it in shell. After 'hack/install-etcd.sh' finishes, it will give an [info](https://github.com/kubernetes/kubernetes/blob/master/hack/lib/etcd.sh#L108) telling developers `export`.

WDYT? @cblecker